toqito/state_metrics/bures_distance.py

```diff
@@ -0,0 +1,67 @@
+"""Bures distance metric."""
+import numpy as np
+
+from toqito.state_metrics import fidelity
+
+def bures_distance(rho_1: np.ndarray, rho_2: np.ndarray ) -> float:
+    r"""
+    Compute the bures distance of two density matrices [WikFid]_.
```

Comment:
`[WikFid]_` should probably have something different like `[WikBures]_`.

Reply:
I'm so sorry. I used the fidelity function as a template. I didn't notice the error. Next time I will write the docs from scratch to avoid this type of mistake.